bundles/model/org.eclipse.smarthome.model.script/src/org/eclipse/smarthome/model/script/actions/Audio.java

```diff
@@ -0,0 +1,128 @@
+/**
+ * Copyright (c) 2014-2016 by the respective copyright holders.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.model.script.actions;
+
+import java.io.IOException;
+
+import org.eclipse.smarthome.core.audio.AudioException;
+import org.eclipse.smarthome.core.library.types.PercentType;
+import org.eclipse.smarthome.model.script.engine.action.ActionDoc;
+import org.eclipse.smarthome.model.script.engine.action.ParamDoc;
+import org.eclipse.smarthome.model.script.internal.engine.action.AudioActionService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The static methods of this class are made available as functions in the scripts.
+ * This allows a script to use audio features.
+ *
+ * @author Kai Kreuzer
+ */
+public class Audio {
+
+    private static final Logger logger = LoggerFactory.getLogger(Audio.class);
+
+    @ActionDoc(text = "plays a sound from the sounds folder to the default sink")
+    static public void playSound(@ParamDoc(name = "filename", text = "the filename with extension") String filename) {
```

Comment:
Note that this was old code that was only moved around.
But I have fixed the order now.